packages/design-system-tokens/src/themes/core.ts

```diff
@@ -76,13 +76,22 @@ export const themeColors: ColorTokens = {
   'muted-inverse':              color['lapis-50'],
   //
   'primary':                    color['ocean-500'],
+  'primary-darker':             color['ocean-700'],
+  'primary-darkest':            color['ocean-800'],
+  // @TODO: deprecate primary-alt as secondary
```

Comment:
Could the references to `primary-alt` be deprecated as part of this work? Or are they being referenced elsewhere still?